apps/api/src/app/notification-template/usecases/update-notification-template/update-notification-template.usecase.ts

```diff
@@ -193,6 +195,15 @@ export class UpdateNotificationTemplate {
       throw new BadRequestException('No properties found for update');
     }
 
+    invalidateCache({
+      service: this.cacheService,
+      storeKeyPrefix: [CacheKeyPrefixEnum.NOTIFICATION_TEMPLATE],
+      credentials: {
+        _id: command.templateId,
+        environmentId: command.environmentId,
+      },
+    });
```

Comment:
Can we extract this to use the invalidate cache decorator instead?

Reply:
I looked at the logic again and we most definitely can! :)